lib/hexpm_web/controllers/dashboard/organization_controller.ex

```diff
@@ -171,7 +175,9 @@ defmodule HexpmWeb.Dashboard.OrganizationController do
       invoice_ids = Enum.map(customer["invoices"], & &1["id"])
 
       if id in invoice_ids do
-        case Hexpm.Billing.pay_invoice(id) do
+        case Hexpm.Billing.pay_invoice(id,
+               audit: %{audit_data: audit_data(conn), organization: organization}
+             ) do
```

Comment:
This looks a bit weird, can you break out something so that we get nicer line breaks?

Reply:
Something like this?

```elixir
        case Hexpm.Billing.pay_invoice(
               id,
               audit: %{
                 audit_data: audit_data(conn),
                 organization: organization
               }
             ) do
```

And there are other places like this one. Update them as well?

Reply:
Maybe:

```elixir
audit = %{audit_data: audit_data(conn), organization: organization}

case Hexpm.Billing.pay_invoice(id, audit: audit) do
```

Reply:
👍 

It's updated in https://github.com/hexpm/hexpm/pull/856/commits/dd07d6aeeadb870b771780d60d96583850322b80, thank you for the suggestion!